plugins/woocommerce-blocks/assets/js/base/context/hooks/use-checkout-address.ts

```diff
@@ -64,6 +67,15 @@ export const useCheckoutAddress = (): CheckoutAddress => {
 		[ setBillingAddress ]
 	);
 
+	const isBillingAddressReadOnly = applyCheckoutFilter( {
+		filterName: 'isBillingAddressReadOnly',
+		defaultValue: false,
+	} );
+	const isShippingAddressReadOnly = applyCheckoutFilter( {
+		filterName: 'isShippingAddressReadOnly',
+		defaultValue: false,
+	} );
+
```

Comment:
I added this in context so that it can easily be shared between the components of Checkout blocks.